src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -121,7 +121,7 @@ if [ ${INCLUDE_DEVELOPER_TOOLS_SERVER} -ne 0 ]; then
         echo "deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ $UBUNTU_CODENAME main" | \
         tee /etc/apt/sources.list.d/kitware.list >/dev/null && \
         apt-get update && \
-        apt-get install -y --no-install-recommends cmake=3.27.7* cmake-data=3.27.7* rapidjson-dev
+        apt-get install -y --no-install-recommends cmake=3.28.* cmake-data=3.28.* rapidjson-dev
```

Comment:
Version 3.28 has a bug with CUDA Toolkit, it was fixed in latest versions.
https://gitlab.kitware.com/cmake/cmake/-/issues/25484